src/client/app/components/HeaderComponent.jsx

```diff
@@ -22,16 +22,21 @@ export default function HeaderComponent(props) {
 	};
 	const loginLinkStyle = {
 		// Displays the login button link only if the user is not logged in or is explicitly told to display by the parent component
-		display: localStorage.getItem('token') || props.renderLoginButton === 'false' ? 'none' : 'inline'
+		display: localStorage.getItem('token') || props.renderLoginButton === false ? 'none' : 'inline'
```

Comment:
I think this ternary operator is hard enough to read that it would be better as a call to a function with an `if-else` block.